### PR TITLE
chore(i18n,docs): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

**Review by sidemt — DISMISSED** [review]
LGTM

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: docs/i18n/japanese/how-to-open-a-pull-request.md [hunk 1]
```diff
@@ -155,28 +155,28 @@ When you are working on features for our upcoming curriculum `next-*` branches,
...
    npm run clean
 
    npm ci
-   npm run test:curriculum --superblock=<superblock-name>
+   npm run test:カリキュラム--superblock=<superblock-name>
 
    # example:
 
-   # npm run test:curriculum --superblock=python-for-everybody
+   # npm run test:カリキュラム-superblock=python-for-everyone
```
[the remark this answers is not in the document]
Comment (ieahleen) [reply docs/i18n/japanese/how-to-open-a-pull-request.md:175]: @sidemt maybe you want to take a look

[the remark this answers is not in the document]
Comment (sidemt) [reply docs/i18n/japanese/how-to-open-a-pull-request.md:175]: @nhcarrigan @ieahleen Thanks. Yes I agree we shouldn't translate this, and I disapproved and fixed it on Crowdin.
Some other commands in this file are translatable on Crowdin. Can we make those not-translatable?

[the remark this answers is not in the document]
Comment (sidemt) [reply docs/i18n/japanese/how-to-open-a-pull-request.md]: Now the fixed translations were downloaded from Crowdin. I'm resolving this conversation.

File: docs/i18n/japanese/how-to-help-with-video-challenges.md [hunk 2]
```diff
@@ -80,61 +80,60 @@ videoId: nVAaxZ34khk
...
 
-In the example above, the URL is https://www.youtube.com/watch?v=nVAaxZ34khk
+上記例では、URL は https://www.youtube.com/watch?v=nVAaxZ34khk です。
 
-Skim the YouTube video with that videoId and think of a multiple-choice question based on the content of the video.
+その videoId でYouTube ビデオに目を通し、ビデオコンテンツに基づいて多肢選択式の質問を考えてください。
 
-### Add the question to the markdown file
+### マークダウンファイルに質問を追加してください
+ローカルでまたは GitHub インターフェースを使用して質問を追加することができます。 ローカルで質問を追加するには、「freeCodeCamp をローカルに設定する」(how-to-setup-freecodecamp-locally.md) 必要があります。 GitHub でファイルを見つけて、編集ボタンをクリックして、ブラウザで質問を追加することもできます。
```
Comment (sidemt) [inline docs/i18n/japanese/how-to-help-with-video-challenges.md:91]: This line does not match with the line number in the source file. I checked Crowdin and the translation was missing a blank line in it. So I added one.
https://freecodecamp.crowdin.com/translate/457d487b1586f10d2fba3c4ccb9c5453/5360/en-ja/106#721946

File: docs/i18n/japanese/how-to-work-on-coding-challenges.md [hunk 3]
```diff
@@ -73,27 +73,27 @@ assert.equal(
 );
 ```
 
-# --notes--
+# -- 注記 --
```
Comment (sidemt) [inline docs/i18n/japanese/how-to-work-on-coding-challenges.md:76]: @nhcarrigan As StephanDue points out in the comment on Crowdin, I think these (`--notes--` `--seed--` etc.) should not be translated. What do you think? 
https://freecodecamp.crowdin.com/translate/457d487b1586f10d2fba3c4ccb9c5453/5352/en-de/134#841821

File: docs/i18n/japanese/how-to-help-with-video-challenges.md (hunk 2 above)
Comment (sidemt) [reply docs/i18n/japanese/how-to-help-with-video-challenges.md:92]: This is fixed now.

**Review by ShaunSHamilton — DISMISSED** [review]
👍

File: docs/i18n/japanese/how-to-work-on-coding-challenges.md (hunk 3 above)
Comment (nhcarrigan) [reply docs/i18n/japanese/how-to-work-on-coding-challenges.md:76]: Correct, these should not be translated. This is part of the parser issue we discussed in chat.

Comment (sidemt) [reply docs/i18n/japanese/how-to-work-on-coding-challenges.md:76]: Understood. Thank you!

**Comment by sidemt** [comment]
There are many Markdown syntax errors in Japanese files. I'm working on fixing them on Crowdin.
I also informed translator/proofreader team working on these files.

**Comment by sidemt** [comment]
I fixed all visible mistakes on Crowdin. Those fixes will be downloaded in the next download.

**Comment by sidemt** [comment]
I went over the files changed today and fixed them on Crowdin.

**Review by naomi-lgbt — APPROVED** [review]
I think this is good to go

**Review by sidemt — APPROVED** [review]
Thanks @nhcarrigan :)
I think the mistakes I fixed today are not critical, so I agree we can merge this now.

**Pull request merged**